drivers/st25r3911b/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+zephyr_library()
+zephyr_library_sources(st25r3911b_spi.c)
+zephyr_library_sources(st25r3911b_common.c)
+zephyr_library_sources(st25r3911b_interrupt.c)
+zephyr_library_sources(st25r3911b_nfca.c)
+zephyr_include_directories(.)
```

Comment:
This directory should not be added to include paths. It is supposed to contain only private header files, isn't it?

Reply:
Yes, this line is no needed more.